lib/resources/platform.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+module Inspec::Resources
+  class PlatformResource < Inspec.resource(1)
+    name 'platform'
+    desc 'Use the platform InSpec resource to test the platform on which the system is running.'
+    example "
+      describe platform.name do
+        it { should eq 'redhat' }
+      end
+
+      describe platform.family?('unix') do
```

Comment:
I wonder if we should try to exploit RSpec's magic treatment of methods with question marks and see if we can't get it to be like this to make it more readable and more similar to existing resources:

```ruby
describe platform do
  it { should be_in_family('unix') }
end
```

That should work if we create a method `def in_family?(family_name)`, right? or am I making that up?